projects/plugins/jetpack/extensions/blocks/coauthor-paragraph/index.js

```diff
@@ -0,0 +1,82 @@
+import { useBlockProps } from '@wordpress/block-editor';
+import { createBlock } from '@wordpress/blocks';
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { getIconColor } from '../../shared/block-icons';
+import attributes from './attributes';
+import edit from './edit';
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+export const name = 'coauthor-paragraph';
+export const title = __( 'Coauthor Paragraph (Experimental)', 'jetpack' );
```

Comment:
I don't know if we need the "Experimental" label if the block is in beta? Do you think it will make sense to keep it as well when the block is available to everyone, since OpenAI itself is kind of in Beta as well?